crates/webidl/src/idl_type.rs

```diff
@@ -490,7 +490,7 @@ impl<'a> IdlType<'a> {
             }
 
             IdlType::Any => dst.push_str("any"),
-            IdlType::Void => dst.push_str("void"),
+            IdlType::Undefined => dst.push_str("undefined"),
```

Comment:
Note this is technically able to change the function names of generated bindings (i.e. a breaking change), but it doesn't happen with any of the WebIDL in this repository.

So this change seems ok unless there's a path where downstream crates are able to provide their own WebIDL, but I don't think we expose that anywhere.

Reply:
Nah seems fine to me!